core-bundle/tests/Contao/EnvironmentTest.php

```diff
@@ -24,13 +24,13 @@ class EnvironmentTest extends TestCase
     /**
      * @var string
      */
-    public $rootDir;
+    public $projectDir;
```

Comment:
not sure it's a problem this is a public property?

Reply:
Since it is a unit test class, it should not be a problem. Not sure why the property is public at all?

Reply:
While we are on it, let's also fix the visibility (see #1892).